raiden/transfer/log.py

```diff
@@ -178,12 +201,21 @@ def log(self, state_change):
         serialized_data = self.serializer.serialize(state_change)
         self.storage.write_transaction(serialized_data)
 
+    def log_events(self, events):
+        """ Log the events into the write ahead Log. They are always assumed to be
+        events coming out of the last logged state change"""
+        assert isinstance(events, list)
+
+        id = self.last_state_change_id()
```

Comment:
This is not safe, we don't know if there was two state changes applied at the same time, we shouldn't rely on implicit orderings, this must have the `state_change_id` as an argument.